### PR TITLE
Tweak new conversation component.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/team-building/index.js [hunk 1]
```diff
@@ -131,6 +157,24 @@ const styles = Styles.styleSheetCreate({
       width: 470,
     },
   }),
+  emptyContainer: Styles.platformStyles({
+    common: {
+      flex: 1,
+    },
+    isElectron: {
+      alignSelf: 'center',
```
Comment (chrisnojima) [inline shared/team-building/index.js]: you can use alignself on the box2 itself, or move this to common

File: shared/team-building/index.js [hunk 2]
```diff
@@ -131,6 +157,24 @@ const styles = Styles.styleSheetCreate({
...
+      paddingBottom: 40,
+    },
+    isMobile: {
+      alignSelf: 'center',
+      maxWidth: '80%',
+      paddingBottom: 150,
+    },
+  }),
+  emptyContainerText: {
+    textAlign: 'center',
```
Comment (chrisnojima) [inline shared/team-building/index.js]: text also has a centered prop for this

File: shared/team-building/shared.js [hunk 3]
```diff
@@ -2,36 +2,54 @@
...
-const _serviceIdToIconFont = {
-  contact: 'iconfont-identity-twitter',
-  facebook: 'iconfont-identity-facebook',
-  github: 'iconfont-identity-github',
-  hackernews: 'iconfont-identity-hn',
-  keybase: 'iconfont-keybase',
-  pgp: 'iconfont-identity-pgp',
-  reddit: 'iconfont-identity-reddit',
-  twitter: 'iconfont-identity-twitter',
+const services = {
```
Comment (chrisnojima) [inline shared/team-building/shared.js]: don't forget this is all going away soon w/ the generic proofs

**Comment by adamjspooner** [comment]
@chrisnojima, this is ready for another look.

File: shared/common-adapters/box.js.flow [hunk 4]
```diff
@@ -5,7 +5,7 @@ export type Props = any
 
 export type Box2Props = {|
   alignItems?: 'center' | 'flex-start' | 'flex-end',
-  alignSelf?: null | 'flex-start' | 'flex-end',
+  alignSelf?: null | 'center' | 'flex-start' | 'flex-end',
```
Comment (chrisnojima) [inline shared/common-adapters/box.js.flow]: i believe we didn't do this cause you're already centered if you're not full width/height

File: shared/team-building/index.js [hunk 5]
```diff
@@ -93,6 +93,7 @@ class TeamBuilding extends React.PureComponent<Props, void> {
           </Kb.Box2>
         ) : !showRecs && !props.showServiceResultCount && !!props.selectedService ? (
           <Kb.Box2
+            alignSelf="center"
```
Comment (chrisnojima) [inline shared/team-building/index.js]: and given the above why does this do anytying if you're full width/height

File: shared/team-building/index.js [hunk 6]
```diff
@@ -162,19 +163,14 @@ const styles = Styles.styleSheetCreate({
       flex: 1,
     },
     isElectron: {
-      alignSelf: 'center',
       maxWidth: 290,
```
Comment (chrisnojima) [inline shared/team-building/index.js]: cause of maxWidth?

Comment (adamjspooner) [reply shared/team-building/index.js]: Yeah. Is there a better way to do this?

**Pull request merged**